hack/jenkins/images/bootkube-e2e/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:xenial
+
```

Comment:
we could do:

```
ARG GOLANG
ENV GOLANG ${GOLANG:-1.10}
ARG TERRAFORM_VERSION
ENV TERRAFORM_VERSION ${TERRAFORM_VERSION:-0.11.3}

Reply:
I'm for hard coding these instead of relying on the developer's env.

Reply:
i'm ok with that as well.

Reply:
Will do asap, thanks, I should use `ARGS` more often.

Reply:
Hm, build args don't directly come from the environment, the dev would have to specify them. That having been said, for things like versions, I think hard-coding is better. (I've found build args to more be useful when using a single Dockerfile with slightly different internal behavior based on a build arg.)

I'm just going to leave it for now. If it's even useful to someone they can send it.